lib/resilient/circuit_breaker.rb

```diff
@@ -17,9 +17,10 @@ def self.reset
     # registered. If key does exist, it returns registered instance instead of
     # allocating a new instance in order to ensure that state/metrics are the
     # same per key.
-    def self.get(key: nil, open: false, properties: nil, metrics: nil, registry: nil)
+    def self.get(key: nil, properties: nil, metrics: nil, registry: nil)
+      key = Key.wrap(key) unless key.nil?
```

Comment:
It doesn't feel great to me to have a `nil` conditional here and a `nil` check+raise in `initialize`. What about teaching `Key.wrap` to return `nil` for `nil` input and letting this be handled in the constructor?